exporter.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+import logging
+from openpyxl import Workbook
+import time
+
+from structlog import wrap_logger
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def create_comments_book(survey_id: str, comments) -> str:
+    logger.info(comments)
```

Comment:
What information can be in the comments? If it can be anything sensitive we shouldn't be logging it